cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java

```diff
@@ -48,6 +50,7 @@
 @ComponentScan({
   "com.netflix.spinnaker.kork.secrets",
   "com.nike.backstopper", // error management TODO explicitly import the config bean
+  "com.nike.cerberus.audit.logger",
```

Comment:
this should be removed and handled in your configuration class.

Reply:
example, your config class is in com.nike.cerberus.config, which is already loaded.

then in your config class you have something like the following 

```java
@Configuration
@ConditionalOnProperty("cerberus.userAuth.connector.okta.enabled")
@ComponentScan({"com.nike.cerberus.auth.connector.okta"})
```

Reply:
- [ ] move @ComponentScan for logger to AthenaAuditLoggerConfiguration